src/prometheus_alert_rules/flapping.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: AlertRules
+  rules:
+  - alert: AlertRulesFlapping
+    expr: changes(ALERTS_FOR_STATE{severity = "warning"}[1h]) > 5
+    for: 5m
```

Comment:
Do we need `for: 5m` if we already have `[1h]` above?

Reply:
In general, i think it's good form to have a pending period before alerts are actually firing.